app/assets/javascripts/show.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() {
+  $('.votes').click(function(event) {
+    event.preventDefault();
+
+    var newVote = this.id.split('-')[0];
+    var rId = this.id.split('-')[1];
+
+    var request = $.ajax({
+      method: "POST",
+      url: '/reviews/' + rId + '/votes',
+      data: { new_vote: newVote, r_id: rId }
```

Comment:
Identifier 'new_vote' is not in camel case.<br>Identifier 'r_id' is not in camel case.